cms/core/logs.py

```diff
@@ -1,23 +1,43 @@
+import logging
+import traceback
 from datetime import datetime
-from logging import LogRecord
 from typing import Any
 
 import json_log_formatter
 from django.conf import settings
 
+SEVERITY_MAPPING = {
+    logging.WARNING: 2,
+    logging.ERROR: 1,
+    logging.CRITICAL: 0,
+}
 
-class JSONFormatter(json_log_formatter.JSONFormatter):
-    """A log formatter which outputs JSON and structures log messages as required."""
-
-    def json_record(
-        self, message: str, extra: dict[str, str | int | float], record: LogRecord
-    ) -> dict[str, str | int | float]:
-        record_data: dict = super().json_record(message, extra, record)
 
-        del record_data["time"]
+class JSONFormatter(json_log_formatter.JSONFormatter):
+    """A log formatter which outputs JSON and structures log messages as required.
+
+    https://github.com/ONSdigital/dp-standards/blob/main/LOGGING_STANDARDS.md
+    """
+
+    def json_record(self, message: str, extra: dict[str, Any], record: logging.LogRecord) -> dict[str, Any]:
+        record_data = {
+            "created_at": datetime.fromtimestamp(record.created),
+            "namespace": record.name,
+            "severity": SEVERITY_MAPPING.get(record.levelno, 3),
```

Comment:
non-blocking suggestion:

```suggestion
            "severity": SEVERITY_MAPPING.get(record.levelno, 3),  # Default to INFO
```